Circuit/Components/VacuumTubes/Pentode.cs

```diff
@@ -93,15 +104,15 @@ public override void Analyze(Analysis Mna)
             var iKoren = Call.If(E1 > 0, Binary.Power(E1, Ex), 0);
             var ip = Call.If(vpk > 0, iKoren / Kg1 * Call.ArcTan(vpk / Kvb), 0);
 
-            var vg = 13d;
-            var knee = 3d;
-            var rg1 = 2000d;
+            var vg = (double)Vg;
```

Comment:
Why cast to double? That's usually not a good idea, because converting base 10 round numbers to doubles, and then to rationals, usually results in much larger rational numerators/denominators than going straight from decimal to rational would.

Reply:
Yeah, I somehow missed the fact that you can explicitly cast `Quantity` to `Real`

Reply:
Done